third_party/move/move-compiler-v2/tests/checking/receiver/vectors.move

```diff
@@ -0,0 +1,20 @@
+// Do not include stdlib for this test:
+// no-stdlib
+
+module 0x1::vector { // must be this module
+
+    fun receiver<T>(self: vector<T>, _y: T) {
```

Comment:
There are currently several methods in the standard vector module which would benefit from receiver syntax (eg, `reverse`). Is the plan to eventually rewrite the standard modules to use `self` where possible (which should be upgrade compatible due to change in only param names, assuming in all applicable cases they were the initial argument)?